vscode-dotnet-runtime-extension/README.md

```diff
@@ -1,16 +1,51 @@
-# .NET Install Tool for Extension Authors
+# .NET Runtime Install Tool
 
- [![Installs](https://ms-dotnettools.gallerycdn.vsassets.io/extensions/ms-dotnettools/vscode-dotnet-runtime/1.6.0/1667237254317/Microsoft.VisualStudio.Services.Icons.Default)](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscode-dotnet-runtime)
+[![Version](https://img.shields.io/visual-studio-marketplace/v/ms-dotnettools.vscode-dotnet-runtime?style=for-the-badge)](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscode-dotnet-runtime) [![Installs](https://img.shields.io/visual-studio-marketplace/i/ms-dotnettools.vscode-dotnet-runtime?style=for-the-badge)](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscode-dotnet-runtime)
 
-This extension allows acquisition of the .NET runtime specifically for Visual Studio Code extension authors. This tool is intended to be leveraged in extensions that are written in .NET and require .NET to boot pieces of the extension (e.g. a language server). The extension is not intended to be used directly by users to install .NET for development.
+This extension provides a unified way for other extensions like the [C#] and [C# Dev Kit] extensions to install local, private versions of the .NET Runtime. This extension is not intended to be used directly by users to install .NET for development purposes because it only includes the .NET Runtime and not the entire .NET SDK.
 
-## Goals: Acquiring .NET for extensions
+## Troubleshooting
```

Comment:
I think we should have an entire section at the very beginning that very crudely responds to the 'Why is this on my machine'? Problem.

I still think some users are not going to come away from this with the understanding that C# or DevKit installed <this> extension for them and that other extensions rely on this one, so that is how it got on their machine, and if they want to remove it, they need to uninstall the extension depending on it (which VS Code will tell them who that is if they try to uninstall.) We should mention that uninstalling the tool should uninstall any runtimes installed by the tool. I would include detail about how this extension is 'required' for the other extensions that depend on it to run